hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -3787,7 +3798,7 @@ private void closeScanner(HRegion region, RegionScanner scanner, String scannerN
       if (region.getCoprocessorHost() != null) {
         region.getCoprocessorHost().postScannerClose(scanner);
       }
-      closedScanners.put(scannerName, scannerName);
+      if (!isError) closedScanners.put(scannerName, rsh.getNextCallSeq());
```

Comment:
The checkstyle config for HBase will issue a warning for this style, please use
```
if (!isError) {
  xxx
}
```

Reply:
Thanks, done